Dockerfile

```diff
@@ -1,69 +1,15 @@
 FROM node:7.9
-RUN apt-get update
```

Comment:
Looks to me like they were removed because git does not need to be installed, since the local clone of the repo is `COPY`d instead (L5).